README.md

```diff
@@ -90,6 +94,10 @@ against the latest version of the SDK on each commit and PR.
 
 #### CentOS
 
+The default compiler on CentOS has poor support for C++11. We need to upgrade
```

Comment:
The default compiler on CentOS doesn't fully support C++11.

Reply:
Fixed.